README.md

```diff
@@ -48,8 +48,14 @@ brew install go
 git clone https://github.com/buildkite/agent.git
 cd agent
 
+# Create the builds directory
+mkdir /tmp/buildkite-builds
+
+# Build the agent
+go build -i -o /usr/local/bin/buildkite-agent .
```

Comment:
Assume you went this way to support `go generate`?

Reply:
That was unrelated to `go generate`.

When testing `buildkite-agent` via the previous example of `go run main.go start` builds failed with the following error:
```
Error: Error building command: exec: "buildkite-agent": executable file not found in $PATH
```
That's why I took the liberty of updating the sample command to build the `buildkite-agent` binary and output it in the `PATH`.

The `go generate` support comes with the comments added here:
https://github.com/buildkite/agent/blob/master/main.go#L3